### PR TITLE
Changes to use lib64 on x86_64

### PR DESCRIPTION
**Pull request by jam7**
Change several packages to use lib64 on x86_64
Also disable making static libraries.

Tested on armv7l and x86_64.

### CODE REVIEW

File: packages/finch.rb [hunk 1]
```diff
@@ -3,34 +3,38 @@
...
+      '--disable-gstreamer-interfaces',
+      '--disable-idn',
+      '--disable-meanwhile',
+      '--disable-screensaver',
+      '--disable-sm',
+      '--disable-tk',
+      '--disable-vv',
+      '--enable-consoleui=yes',
+      '--includedir=/usr/local/include',
+      '--oldincludedir=/usr/local/include',
```
Comment (uberhacker) [inline packages/finch.rb]:
```
"--includedir=#{CREW_PREFIX}/include",
"--oldincludedir=#{CREW_PREFIX}/include",
```
Better?

**Comment by uberhacker** [comment]
I'm wondering if we should just automatically add the following configure options to every package:
```
"--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}", "--includedir=#{CREW_PREFIX}/include"
```

File: packages/finch.rb (hunk 1 above)
Comment (jam7) [inline packages/finch.rb]: done

**Comment by jam7** [comment]
@uberhacker Nice idea.  Why don't you try that?
@flyingP0tat0 BTW, x11 libraries looks like CREW_LIB_PREFIX is needed also.

**Comment by thedamian** [comment]
I hate updates that "do it right" 

Is it right if it's already working? What HUGE benefits do we get with these changes? I'm all for progress but don't want to screw up anything while doing that.

**Comment by jam7** [comment]
It's worth to do for only Chrome OS 60 or above on x86_64.  Chrome OS 60 is in beta channel.  It will arrives stable channel soon.  This PR is preparing for it.

**Pull request merged**